.github/workflows/ci.yml

```diff
@@ -111,26 +111,29 @@ jobs:
       matrix:
         python-version: [ 3.9 ]
         os: [ "ubuntu-latest" ]
+        environment: [ "legacy-production" ]
```

Comment:
@rathishcholarajan can you setup an environment (`legacy-production`) along with the required environment configurations (`QISKIT_IBM_TOKEN`, `QISKIT_IBM_URL`, `QISKIT_IBM_INSTANCE`)?

Ideally also for `legacy-staging`. Will enable a nightly run against staging as soon as the tests against production run stable.

Reply:
Done both